docs/content/topics/operators/security.md

```diff
@@ -19,28 +19,35 @@ For that reason, it is prudent to configure security.
   in a Helm install, do `helm install --namespace brigade ...`.
 - *RBAC Enabled*: When installing with Helm, role-based access control is enabled by default.
   To disable,`--set rbac.enabled=false` will turn off role-based access control.
-- *Do not run more than one brigade per namespace*: Running multiple installs of Brigade
+- *Do not run more than one Brigade per namespace*: Running multiple installs of Brigade
   in the same namespace can cause naming collisions which could result in
   unauthorized access to pods.
```

Comment:
I don't think this is an issue anymore on account of every project getting its own namespace, although I cannot say I have tried it.

Reply:
Ah, right, just checked assumptions here.  As it stands currently, it is only possible for one Brigade instance to deploy (and function correctly) per cluster.  I'll add a note around this.  We can always revise if/when we support multiple instances on a cluster.

Reply:
Actually, I removed this note for now, since we now have https://github.com/brigadecore/brigade/issues/1630